protocol/x/perpetuals/client/cli/query.go

```diff
@@ -30,6 +30,7 @@ func GetQueryCmd(queryRoute string) *cobra.Command {
 	cmd.AddCommand(CmdQueryPremiumSamples())
 	cmd.AddCommand(CmdQueryPremiumVotes())
 	cmd.AddCommand(CmdQueryAllLiquidityTiers())
+	cmd.AddCommand(CmdModuleNameToAddress())
```

Comment:
This gives you the bytes but not the `dydx` prefixes address - is that intended?

```
build/dydxprotocold query perpetuals module-name-to-address subaccounts
61CF88998531191CE56D33D6CC6C1EE4D11691C1
```

Reply:
No, it's supposed to give you the dydx address. Something got screwed up in my cleanup. Looking now

Reply:
Also a bit weird to put `x/perpetuals` but recognize we don't have a good module for this. `govplus` was meant to be a catchall module but this isn't directly related to governance either. 

If we do keep this in `x/perpetuals` for now, can we create a ticket to move this to a better place later

Reply:
I agree x/perpetuals is a weird place but I also couldn't find a generic module for this. Do you have a preference for one?

Reply:
Ok maybe `x/govplus` for now since it's the closest thing to a catchall module

Reply:
Why is the governance module a catchall module?

Reply:
Non-blocking, but this command does not relate to a module so if we wanted it to be in a more logical place, it should just be added under the root command [here](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/cmd/dydxprotocold/cmd/root.go).